modules/ROOT/pages/apikit-4-xml-reference.adoc

```diff
@@ -1,91 +1,97 @@
 
-This attribute is deprecated since APIkit 1.2.0 and only kept for compatibility with APIkit 1.1.x. +
-It's recommended that API implementations use "api" instead of “raml”.
-|*parser* | N | Specifies the parser used to validate requests.
+This attribute is deprecated since APIkit 1.2.0. Included to provide compatibility with APIkit 1.1.x. +
+Use `api` instead of `raml` in your API implementations.
+|parser | N |Parser used to validate requests.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
 
 Possible values are:
 
-* AUTO: The APIkit Router will try to initiate using AMF, it's not possible (it has errors), then it will try with the RAML Parser.
-* AMF: The APIkit Router will use AMF to read the API and validate incoming requests.
-* RAML: The APIkit Router will use RAML Parser to read the API and validate incoming requests. This is similar to previous APIkit versions.
+* AUTO
++
+*APIkit Router* initiates by using AMF. If it has errors, it initiates by using the RAML parser.
+* AMF
++
+*APIkit Router* uses AMF to read the API and validate incoming requests.
+* RAML
++
+*APIkit Router* uses RAML parser to read the API and validate incoming requests. Similar to previous APIkit versions.
 
-*Default Value: AUTO*.
+Default Value: AUTO.
 
-| *api* | N | Root API file location.
+| api | N | Root API file location.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
 
-It can be any of a URL, the relative path to a file located under `src/main/resources/api`, or the API Sync resource path.
+It can be any URL, the relative path to a file located under `src/main/resources/api`, or the `API Sync` resource path.
 
-| *keepApiBaseUri* | N | *This attribute is valid for APIkit 1.2.0 and later*
-
-When set to *false*, it rewrites the `baseUri` in the RAML definition of the API. +
-If set to *true* it does not rewrite the `baseUri` field.
+| `keepApiBaseUri` | N | Attribute valid for APIkit 1.2.0 and later.
 
+When set to `false`, it rewrites the `baseUri` in the RAML definition of the API. +
+If set to `true`, it does not rewrite the `baseUri` field.
 
 |===
 
-If you are using APIkit behind a reverse proxy, you can use API Console if you set `keepRamlBaseUri` to true to put the URL of a proxy in the baseUri property.
+If you use APIkit behind a reverse proxy and set `keepRamlBaseUri` to `true`, use *API Console* to put the URL of a proxy in the `baseUri` property.
 
-=== apikit:router Attributes
+=== `apikit:router` Attributes
 
 [%header,cols="22a,10a,68a"]
 |===
 |Attribute Name|Required |Description
-|*disableValidations* |N |Disables validations of requests.
-|*headerStrictValidation* |N |Validates that request headers are defined in RAML file or belong to a well-known set of headers.
-|*queryParamsStrictValidation* |N |Validates that request query parameters are defined in RAML file.
+|`disableValidations` |N |Disables validations of requests.
```

Comment:
No periods if none are complete sentences.

Reply:
Lines 95 and 96 are complete sentences. That's the reason why I added the periods.
@hannanelson, please, let me know if I missed something. 
Thanks!

Reply:
What is the subject of those sentences on 95 and 96?

Reply:
The subject of these sentences is the attribute name, mentioned in the first column. 
I added the subjects. Thanks!

Reply:
OK, I don't think that counts as part of the sentence in the third column. The contents of a cell must be a full sentence for it to be considered a full sentence. 

It's fine to have it the way you do as long as you remove the period from the sentence fragments in the third column.

Reply:
Here's what the Style Guide says about punctuation in tables:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-tables
Specifically:
Make column cell entries grammatically and structurally parallel. End text with periods unless all entries in any one column are a word or other fragment.

Reply:
Hi, @hannanelson! When I read the style guide I follow the rules and added the periods in the third column because they were full sentences, though they did not have a subject. 
Thanks!